lib/src/commands/list.dart

```diff
@@ -44,6 +45,10 @@ class ListNodeQueryProcessor extends QueryProcessor {
       enableActions = true;
     }
 
+    if (stream.getAttribute("option:enableCache") == false) {
```

Comment:
I think these could be initialized with ternary operators and turned into `final`

Reply:
These attributes can change. But I'll give it a look.